packages/api/amplify_api/example/integration_test/graphql/api_key_test.dart

```diff
@@ -27,28 +24,15 @@ void main({bool useExistingTestUser = false}) {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   group('GraphQL API key', () {
-    // TODO(ragingsquirrel3): get apiKey from config until multiauth param
-    final amplifyConfig = AmplifyConfig.fromJson(
-      json.decode(amplifyconfig) as Map<String, dynamic>,
-    );
-    final gqlEndpoint = amplifyConfig.api!.awsPlugin!.entries
-        .firstWhere(
-          (element) => element.value.endpointType == EndpointType.graphQL,
-        )
-        .value;
-    final apiKey = gqlEndpoint.apiKey!;
-
     setUpAll(() async {
       await configureAmplify();
       await signOutTestUser();
     });
 
     group('queries', () {
       testWidgets('should query authorized model', (WidgetTester tester) async {
-        final req = authorizeRequestForApiKey(
-          ModelQueries.list<Blog>(Blog.classType),
-          apiKey,
-        );
+        final req =
+            authorizeRequestForApiKey(ModelQueries.list<Blog>(Blog.classType));
```

Comment:
*nit:* Might want to update the lint rules if "library" from amplify_lints isn't already being used.
```suggestion
        final req = authorizeRequestForApiKey(
            ModelQueries.list<Blog>(Blog.classType),
        );
```

Reply:
plan to do as separate PR bc anticipating lots of related changes